Gemfile.lock

```diff
@@ -66,6 +67,8 @@ GEM
     safe_yaml (1.0.5)
     sass-embedded (1.67.0-arm64-darwin)
       google-protobuf (~> 3.23)
+    sass-embedded (1.67.0-x64-mingw-ucrt)
```

Comment:
This change is not related to the commit of reverting the files. Hopefully it will go away after a rebase.